autopts/utils.py

```diff
@@ -457,6 +457,14 @@ def exit_if_admin():
         sys.exit("Administrator rights are not required to run this script!")
 
 
+def log_memory_usage():
+    process = psutil.Process(os.getpid())
+    mem_info = process.memory_info()
+
+    mem_usage = mem_info.rss / (1024 ** 2)
+    logging.debug(f"Memory usage: {mem_usage:.2f} MB")
```

Comment:
Function defined but never called?

Reply:
Yes, you can use this if you need to debug a memory leak in one of the autopts scripts.

Reply:
Right, so you have to make source code modifications in order to use this? 

Should this be possible to enable/log automatically via a CLI option?

Reply:
> Right, so you have to make source code modifications in order to use this?

Yes.

> Should this be possible to enable/log automatically via a CLI option?

Might be useful.